SDALib-CPP/SDALibTests/Mocks/PythonDriverMock.h

```diff
@@ -0,0 +1,33 @@
+#pragma once
+#include "SDADriver.hpp"
+#include "SDADriver.inl"
+#include "PythonDriver.h"
+#include "SDADriverMock.h"
+#include "PointerManagerMock.h"
+
+/// @brief A python driver mock
+class PythonDriverMock : public AIInterface<PointerManagerMock>
+{
+public:
+    PythonDriverMock()
+        : AIInterface<PointerManagerMock>()
+    {
+        // do nothing
+    }
+
+protected:
+    void InitAI() override
+    {
+        // do nothing
+    }
+
+    SDAAction UpdateAI(SDAData& p_data) override
+    {
+        SDAAction action;
+        action.Steer = 0.75;
+        action.Brake = 2;
+        action.Gear = 1;
+        action.Accel = 0.5;
+        return action;
```

Comment:
Would be better to make these variables that you can set in InitAI and then you can test with random values instead of hard-codes ones. (Obviously you have to change the tests as well then)